src/containers/collectibles/helpers.ts

```diff
@@ -88,13 +82,7 @@ export const assetToCollectible = async (
   let gifUrl = null
```

Comment:
Could use a larger doc-string style comment on `assetToCollectible` that explains our algo here, basically one that explains in a bit more detail why we check extensions and then check mime type (for the case where there is no extension)

Reply:
yes!